### PR TITLE
Add Damn Vulnerable GraphQL Application

### PR DESCRIPTION
**Pull request by dolevf**
Hi! 

Would love to include my new damn-vulnerable project for GraphQL (DVGA)

### CODE REVIEW

File: src/data/offline.json [hunk 1]
```diff
@@ -1188,14 +1188,30 @@
...
 				"name": "guide",
 				"url": "https://owasp-skf.gitbook.io/asvs-write-ups/"
 			}
 		],
-		"author": "glenn.ten.cate@owasp.org and riccardo.ten.cate@owasp.org",
-		"notes": "Check the guide URL here you find how to pull the Docker images and run them locally.",
-		"badge": "blabla1337/skf-labs"
+		"author": "Dolev Farhi <dolev@lethalbit.com>",
+		"notes": null,
+		"badge": null
```
Comment (kingthorin) [inline src/data/offline.json:1215]:
```suggestion
		"badge": "dolevf/Damn-Vulnerable-GraphQL-Application"

```

**Review by kingthorin — CHANGES_REQUESTED** [review]
Is there a reason the new app references the skf guide?

**Comment by dolevf** [comment]
Nope. Fat fingers :)

**Comment by dolevf** [comment]
Should be alright! Thanks for making this repository available to the community.

**Review by dolevf — COMMENTED** [review]
LGTM

File: src/data/offline.json [hunk 2]
```diff
@@ -1189,13 +1189,25 @@
 			"Ruby"
 		],
 		"references": [
-			{
-				"name": "guide",
-				"url": "https://owasp-skf.gitbook.io/asvs-write-ups/"
-			}
```
Comment (kingthorin) [inline src/data/offline.json:1195]: SKF should keep its reference

**Review by kingthorin — APPROVED** [review]
Thanks

**Comment by kingthorin** [comment]
Deployed In https://github.com/OWASP/www-project-vulnerable-web-applications-directory/pull/39

**Pull request merged**